README.md

```diff
@@ -1,23 +1,150 @@
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* How to run the test suite
+# 📖 [Hellow-World] <a name="about-project"></a>
+This is hello world  application
+> .
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><p> <b>Ruby: </b> The primary programming language for the project.</p></li>
+    <li><p><b>Rails: </b> It is a ruby frameworks.</p></li>
+    <li><p><b>React: </b> It is a javacsript frameworks.</p></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>none</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+</ul>
+
+<h1>🚀 Live Demo</h1>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+Coming soon.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Computer (MAC or PC)
+- Code editor (VSCode,Atom etc...)
+- A browser (Chrome,Mozilla,Safari etc...)
+- Version Control System (Git and Github)
+- Rails installed: To run a Rails project, users must have Rails installed on their system. You can include the minimum required version of Rails needed to run the project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:david30-maker/hello-world-project.git
+```
+
+### Install
+
+Install this project with:
+
+cd 'project name'
+
+"npm i" to install all dependencies
+
+"bundle install" to install all gems
+
+"rails s" to run the server
+
+### Run tests
+
+"bundle exec rspec" to run the tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Okpah Igbo**
+
+- GitHub: [@david30-maker](https://github.com/david30-maker)
+- Twitter: [@davidigbo1](https://twitter.com/davidigbo1)
+- LinkedIn: [davidigbo/](https://www.linkedin.com/in/davidigbo/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project rate it and give it a star...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE.md) licensed.
```

Comment:
- [ ] The license link is still not fixed. The file you created needs to be properly linked.

  ![image](https://github.com/david30-maker/hello-world-project/assets/81468675/bd8f92ed-3979-40d5-8d76-95e79bb570dd)